test/lit/passes/optimize-instructions-gc-tnh.wast

```diff
@@ -523,17 +520,14 @@
   ;; TNH-NEXT:  )
   ;; TNH-NEXT: )
   ;; NO_TNH:      (func $null.arm.null.effects (type $void)
-  ;; NO_TNH-NEXT:  (block ;; (replaces unreachable StructSet we can't emit)
+  ;; NO_TNH-NEXT:  (block
```

Comment:
The select propagated unreachability earlier, I suppose, leading to these (positive) changes?